ui/lib/core/addon/components/stat-text.js

```diff
@@ -4,10 +4,10 @@
  *
  * @example
  * ```js
- * <StatText @label="Active Clients" @stat="4,198" @size="l" @subText="These are the active client counts"/>
+ * <StatText @label="Active Clients" @value="4,198" @size="l" @subText="These are the active client counts"/>
  * ```
  * @param {string} label=null - The label for the statistic
- * @param {string} value=null - Value passed in, usually a number or statistic
+ * @param {integer} value=null - Value passed in, usually a number or statistic
```

Comment:
I think the standard jsdoc param type is `number`. I think it's fine if we want to use `integer` but wanted to point that out.

Reply:
Ohh, I'll fix that! Thank you!